worker/src/lib.rs

```diff
@@ -205,9 +171,9 @@ pub async fn main(req: Request, env: Env, _ctx: worker::Context) -> Result<Respo
 
     let sessions = SessionAllocator::new(Arc::new(string_store));
 
-    // if req.headers().get("upgrade").unwrap() == Some("websocket".to_string()) {
-    //     return linkup_ws_handler(req, sessions).await;
-    // }
+    if req.headers().get("upgrade").unwrap() == Some("websocket".to_string()) {
+        return linkup_ws_handler(req, sessions).await;
+    }
```

Comment:
**_suggestion:_**
Maybe to avoid `.unwrap()` we can do something like this? Wdyt?
```suggestion
    if let Ok(Some(upgrade)) = req.headers().get("upgrade") {
        if upgrade.as_str() == "websocket" {
            return linkup_ws_handler(req, sessions).await;
        }
    }
```